### PR TITLE
Adds wrenching/unwrenching to some machines.

### PR DESCRIPTION
**Pull request by Birdtalon**
Note; I need to rework this to use a pre-existing proc which I discovered after opening this PR. Please hold.

Adds unwrenching/wrenching to some machines so they can be moved about.

- Chem Master
- Chem dispenser
- Book binder
- Library computers

Maintainers can you please let me know how many fixes I must cite for this?

🆑 Birdtalon
add: Ability to wrench and move the library computers, chem master, chem dispenser and book binder.
/🆑

### CODE REVIEW

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 1]
```diff
@@ -193,6 +193,9 @@
 	return 1 // update UIs attached to this object
 
 /obj/machinery/chem_dispenser/attackby(obj/item/reagent_containers/B, mob/user, params)
+	if(!anchored)
+		to_chat(user, "<span class='notice'>You must anchor [src] first!</span>")
+		return
 	if(isrobot(user))
```
Comment (farie82) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm]: Robots will get the false hope now that they can interact with it once anchored. Evil man

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 2]
```diff
@@ -251,6 +265,9 @@
 /obj/machinery/chem_dispenser/attack_hand(mob/user)
 	if(stat & BROKEN)
 		return
+	if(!anchored)
+		to_chat(user, "<span class='notice'>You must anchor [src] first!!</span>")
```
Comment (farie82) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm]: Double exclamation mark. Double the need.

**Comment by DesolateG** [comment]
What would be the point of the portable chem dispenser if you can unwrench and move a chem dispenser around?

**Comment by Birdtalon** [comment]
@farie82 You raise a good point and I looked into it further and found there actually is already a proc for this defined on `/obj/` level. So I need to rework this to use it which I will do soon.

@DesolateG the portable one can be constructed whereas this one cannot. However I still think it's worth being able to move these machines since the only way of removal currently is with a bomb.

**Comment by Birdtalon** [comment]
Closing for now.

**Pull request closed without merging**